testing/sdk_plan.py

```diff
@@ -201,13 +203,22 @@ def get_plans_on_failure(request):
     """
     yield
     if sdk_utils.is_test_failure(request):
-        for plan_name in list_plans(framework_name):
-            plan = get_plan(framework_name, plan_name)
-            if not plan:
-                log.error('Unable to fetch {} plan for {}'.format(framework_name, plan_name))
-                continue
-            out_path = os.path.join(
-                sdk_utils.get_test_log_directory(request.node),
-                '{}_plan.txt'.format(plan_name))
-            with open(out_path, 'w') as f:
-                f.write(json.dumps(plan, indent=2))
+        try:
+            log.info('Fetching plans from {}...'.format(framework_name))
```

Comment:
Since we don't have an "official" linter, it's not critical, but we could consider using `%s` for log messages instead of `{}`.